ush/err_exit

```diff
@@ -58,8 +58,10 @@ if [ "$SENDECF" = "YES" ]; then
 fi
 
 # KILL THE JOB:
-if [ -z $PBS_JOBID ]; then
-ecflow_client --kill=${ECF_NAME:?}
+if [ ! -z $PBS_JOBID ]; then
+  qdel $PBS_JOBID
+elif [ ! -z $SLURM_JOB_ID ]; then
+  scancel $SLURM_JOB_ID
```

Comment:
I think this needs to be reworked in the context of `SENDECF`.  

A suggestion would be:
```suggestion
if [ "$SENDECF" = "YES" ]; then
  ecflow_client --kill=${ECF_NAME:?}
fi

if [ ! -z $PBS_JOBID ]; then
  qdel $PBS_JOBID
elif [ ! -z $SLURM_JOB_ID ]; then
  scancel $SLURM_JOB_ID
fi
```

This will ensure that `ecflow` is killing the job with the name `ECF_NAME` and any remaining zombie processes are being cleaned out by the appropriate scheduler command `qdel` or `scancel`.
A `sleep 10` may be necessary before invoking the scheduler commands, but that remains to be tested.